anndata/_core/views.py

```diff
@@ -120,6 +120,14 @@ def drop(self, *args, inplace: bool = False, **kw):
             df.drop(*args, inplace=True, **kw)
 
 
+class AwkwardArrayView(_ViewMixin, AwkArray):
+    def copy(self, order: str = "C") -> np.ndarray:
+        import awkward._v2 as ak
+
+        # we want a copy of an akward array
+        return ak.copy(self)
```

Comment:
as awkward arrays are immutable, would `return self` be enough here?

Reply:
It should be fine. You only want to make copies if you cast as NumPy, modify in place, and then cast back as Awkward. It's something that you'd be doing deliberately and would know about. None of the functions in the Awkward library modify in place.